sdk/identity/identity/src/msal/utils.browser.ts

```diff
@@ -0,0 +1,294 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
can we file an issue to break apart this file better so the browser and node differences are a little more clear? I realize part of the struggle is which package we're importing the types from, though I don't think every method actually is impacted

Reply:
This is what we've done for 4.0.0, but we should revisit this